themes/conventional-commits/static/css/scss/layout/_welcome.scss

```diff
@@ -41,13 +45,13 @@
     color: $welcome-action-color;
     border: 1px solid $welcome-action-color;
     margin-right: $gap-md;
-    transition: background-color .3s ease, color .3s ease;
+    transition: background-color 0.3s ease, color 0.3s ease;
     &:last-child {
       margin-right: 0;
     }
     &:hover {
       background-color: $welcome-action-color;
-      color: get-color-accessible-for-background($welcome-action-color)
+      color: get-color-accessible-for-background($welcome-action-color);
```

Comment:
This was my `Prettier` - I can roll back if requested.

Reply:
Yes please, let the style as it is, edit just what needed 😄